src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  --color: #00acdc;
+  --default_color: #000;
+}
+
+.header {
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  display: inline-block;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+}
+
+.header__container {
+  max-width: 1200px;
+  display: flex;
+  flex-direction: row;
+  margin: 0 auto;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  position: relative;
+  color: var(--default_color);
+  font-style: normal;
+}
+
+.nav__link:hover {
+  color: var(--color);
+}
+
+.is-active {
```

Comment:
you can shorten your code a bit by moving your selector to nav__link:hover separated by commas (.nav__link:hover, .is-active), because both of selector have the same rule

Reply:
got it, will do